internal/engine/image_build_and_deployer.go

```diff
@@ -391,6 +392,13 @@ func injectImageDependencies(iTarget model.ImageTarget, iTargetMap map[model.Tar
 		return model.ImageTarget{}, errors.Wrap(err, "injectImageDependencies")
 	}
 
+	// HACK(maia): until buildkit/parse supports `# syntax = ` flags, check whether the original
```

Comment:
this feels like the wrong place to put this code. Can you move this into the ast and Print() function?

Reply:
fair, will do